README.md

```diff
@@ -95,10 +103,10 @@ SSH_KEY_NAME=your_ssh_key_name_here
 make build-cli
 
 # Create infrastructure using a JSON file
-talis infra create -f create.json
+talis infra create -f create.json_example
 
 # Delete infrastructure using a JSON file
-talis infra delete -f delete.json
+talis infra delete -f delete.json_example
```

Comment:
So we can update this some more. 

The `_example` json files should be examples. The steps should be to make a copy of the `create.json_example` as `create.json` and modify as needed. 

Then the cli command `talis infra create` will now auto generate a delete json file based on the successful create request.

Reply:
try it out, it's slick :-)

Reply:
I don't see any new commits.